src/routes/about.tsx

```diff
@@ -0,0 +1,109 @@
+import Page from '@/components/page'
+import { css } from 'linaria'
+
+const pageStyle = css`
+  text-align: center;
+  display: grid;
+  grid-template-columns: 1fr minmax(60%, 61rem) 1fr;
+  padding: 0 1rem;
+`
+
+const headerStyle = css`
+  grid-column: 2;
+  margin-bottom: 0rem;
+`
+
+const informationStyle = css`
+  font-size: 22px;
+  grid-column: 2;
+`
+
+const sourceStyle = css`
+  padding: 1rem 2rem 0 2rem;
+  font-family: monospace;
+  grid-column: 2;
+`
```

Comment:
Out of curiosity, why does the text switch to monospace here? I think it might be suitable to leave the font as-is in the text that links to github.

Reply:
I thought it looked cooler. Perhaps that's a bad reason.

Reply:
Personally, I am not a huge fan, but that's subjective :smile: 

[Here's an article that talks about how it's beneficial to use a consistent font throughout a design](https://elearningindustry.com/role-of-typography-in-web-design-tips-and-best-practices). In my opinion, since both the list of github repos and the text above it would both be considered "body text", I think they should be the same font.

Take that as you will, not trying to force your hand here :wink: